test/util_mjolnir.cc

```diff
@@ -0,0 +1,100 @@
+#include <fstream>
+#include <sstream>
+#include <unordered_map>
+
+#include <boost/property_tree/ptree.hpp>
+
+#include "baldr/graphid.h"
+#include "baldr/rapidjson_utils.h"
+#include "filesystem.h"
+#include "mjolnir/util.h"
+
+#include "test.h"
+
+#if !defined(VALHALLA_SOURCE_DIR)
+#define VALHALLA_SOURCE_DIR
+#endif
+
+#if !defined(VALHALLA_BINARY_DIR)
+#define VALHALLA_BINARY_DIR
+#endif
+
+namespace {
+using boost::property_tree::ptree;
+using valhalla::baldr::GraphId;
+using valhalla::mjolnir::build_tile_set;
+using valhalla::mjolnir::TileManifest;
+using namespace valhalla;
+using namespace valhalla::midgard;
+
+// Verify that this function runs
+TEST(UtilMjolnir, BuildTileSet) {
+  ptree config;
+  const std::string tile_dir("test/data/util_mjolnir_test_tiles");
+  config.put<std::string>("mjolnir.tile_dir", tile_dir);
+  config.put<unsigned long>("mjolnir.id_table_size", 1000);
```

Comment:
`2020/08/26 12:34:22.512832 [INFO] Building 4 tiles with 36 threads... `
Here's what I noticed while debugging through ssh. That might cause memory issues/bad_alloc. Though ci isn't red at the moment..

Reply:
**RIGHT**. Ok, we can add an explicit configuration here to make sure that it's single threaded.

Reply:
yep set `concurrency: 1` in the config and you shuld be good!